packages/dataviews/src/view-list.tsx

```diff
@@ -2,6 +2,9 @@
  * External dependencies
  */
 import clsx from 'clsx';
+// Import CompositeStore type, which is not exported from @wordpress/components.
+// eslint-disable-next-line no-restricted-imports
+import type { CompositeStore } from '@ariakit/react';
```

Comment:
cc @WordPress/gutenberg-components I presume we should re-export the type from our components package?